routes/definitions.js

```diff
@@ -26,12 +26,14 @@ async function getDefinition(request, response) {
   const log = logger()
   log.info('getDefinition route hit', { ts: new Date().toISOString(), requestParams: request.params })
 
+  let coordinates
+
   // Painful way of handling go namespaces with multiple slashes
   // Unfortunately, it seems the best option without doing a massive
   // rearchitecture of the entire coordinate system
-  if (request.params.type == "go" && request.params.provider == "golang") {
+  if (request.params.type == 'go' && request.params.provider == 'golang') {
```

Comment:
Possibly some lint rule is not setup very well.

Reply:
Michael, do you mean that these should be triple equals to be the best JavaScript it can be? `===`
(Agree if so)

Reply:
Good catch. the == should be ===. What I mean is the double quote can be caught by lint rules. I think == could also be caught in lint rules.